app/models/setting.rb

```diff
@@ -201,8 +201,7 @@ def self.default_settings
 
   def self.load_defaults
     return false unless table_exists?
-    Foreman::Deprecation.deprecation_warning('3.4', "subclassing Setting is deprecated '#{name}' should be migrated to setting DSL "\
```

Comment:
This only removes the deprecation. Perhaps @ezr-ondrej can enlighten us, but I think the proper fix is to remove the whole method.